com.avaloq.tools.ddk.workflow/src/com/avaloq/tools/ddk/workflow/GenerateExport.mwe2

```diff
@@ -16,103 +16,13 @@ import org.eclipse.xtext.ui.generator.*
-        directory = "${runtimeProject}.ui/src-gen"
-    }
-
-    component = Generator {
-        pathRtProject = runtimeProject
-        pathUiProject = "${runtimeProject}.ui"
-        projectNameRt = projectName
-        projectNameUi = "${projectName}.ui"
-        language = {
-            uri = grammarURI
-            fileExtensions = file.extensions
-
-            // Java API to access grammar elements (required by several other fragments)
-            fragment = grammarAccess.GrammarAccessFragment {}
-
-            // Generate java classes for Ecore models
-            fragment = ecore.CustomClassEcoreGeneratorFragment auto-inject {}
-
-            // the serialization component
-            fragment = serializer.SerializerFragment {}
-
-            // a custom ResourceFactory for use with EMF
-            fragment = resourceFactory.ResourceFactoryFragment {
-                fileExtensions = file.extensions
-            }
-
-            // the following fragment generates an Antlr parser -->
-            fragment = parser.antlr.XtextAntlrGeneratorFragment {
-            }
-
-            // java-based API for validation
-            fragment = validation.JavaValidatorFragment {
-            // composedCheck = "org.eclipse.xtext.validation.ImportUriValidator"
-            // composedCheck = "org.eclipse.xtext.validation.NamesAreUniqueValidator"
-            // registerForImportedPackages = true
-            }
-
-            // scoping and exporting API
-            fragment = scoping.ImportNamespacesScopingFragment {}
-            fragment = exporting.SimpleNamesFragment {}
-            fragment = org.eclipse.xtext.generator.builder.BuilderIntegrationFragment {}
-
-            // formatter API
-            fragment = org.eclipse.xtext.generator.formatting.FormatterFragment {}
-
-            // labeling API
-            fragment = labeling.LabelProviderFragment {}
-
-            // outline API
-            fragment = outline.OutlineTreeProviderFragment {}
-            fragment = outline.QuickOutlineFragment {}
-
-            // quickfix API
-            fragment = quickfix.QuickfixProviderFragment {}
-
-            // content assist API
-            fragment = contentAssist.JavaBasedContentAssistFragment {}
-
-            // generates a more lightweight Antlr parser and lexer tailored for content assist
-            fragment = parser.antlr.XtextAntlrUiGeneratorFragment {
-                options = {
-                    backtrack = true // see above
-                }
-            }
-
-            // rename refactoring
-            // fragment = refactoring.RefactorElementNameFragment {}
-
-            // Code generator
-            fragment = generator.GeneratorFragment {}
-
-            // provides a compare view
-            fragment = compare.CompareFragment {
-                fileExtensions = file.extensions
-            }
-
-        }
-    }
+@com.avaloq.tools.ddk.workflow.CommonXbase auto-inject {
+    languageName = "CheckCfg"

```

Comment:
languageName should be "Export" here no?

Reply:
Right, this mwe2 file was wrongly, it does not use the CommonXbase. I have modified it now and left it standalone.